frontend/tasks/content.js

```diff
@@ -1,244 +0,0 @@
-const gulp = require("gulp");
```

Comment:
Everything in here has been scooped out and re-organized into simpler content transform functions in modules under `frontend/lib/content`